src/api/unified/symbol_manager.cpp

```diff
@@ -230,6 +238,71 @@ af_err AFSymbolManager::setBackend(af::Backend bknd) {
     }
 }
 
+af_err AFSymbolManager::addBackendLibrary(const char *lib_path) {
+    if ((numBackendHandles + 1) > MAX_BKND_HANDLES) {
+        // No more space for an additional handle
+        UNIFIED_ERROR_LOAD_LIB();
```

Comment:
You should add a better error message here. This is not descriptive enough.

Reply:
Did you mean the comment? Or the way the error is reported? I'm actually kind of wary also of using `UNIFIED_ERROR_LOAD_LIB` again and again in every error scenario, if that's your concern.